backend-tests/tests/test_azure.py

```diff
@@ -117,9 +117,19 @@ def check_integrations(self, user: User, expected_integration: Dict):
         # Check for equality by parts:
         # Check that actual properties are a subset of expected integrations
         for part in actual.split(";"):
+            # SharedAccessKey will be masked, with only the first 4 characters visible
+            # and the rest of the string replaced with a place holder. For this reason,
+            # we'll test the first 20 bytes only
+            if part.startswith('SharedAccessKey='):
+                part = part[:20]
             assert part in expected_integration["credentials"]["connection_string"]
```

Comment:
Doesn't both sides of the assertion require truncating the access key?

Reply:
no, I don't think so. The portion we take from part should be contained in the string.

Reply:
Right! I thought that the expected value was a list of the connection string key/value pairs.